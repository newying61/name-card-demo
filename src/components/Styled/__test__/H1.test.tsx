import React from 'react';
import { render, cleanup, fireEvent } from '@testing-library/react';
import { H1 } from '../';

afterEach(cleanup);

describe('H1', () => {
  it('should render without crash', () => {
    const { container } = render(
      <H1>Test</H1>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});