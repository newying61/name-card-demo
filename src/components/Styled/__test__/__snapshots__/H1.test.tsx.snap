// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 should render without crash 1`] = `
<h1
  class="sc-htpNat gWfLTp"
>
  Test
</h1>
`;
